summary_workflows/differential_usage/specification/D_benchmark_specification.md

```diff
@@ -0,0 +1,133 @@
+
+Comparison of differential PAS usage predicted from simulated RNA-Seq data with dataset used for simulation
+
+1. List of genes with information on differentially used poly(A) sites identified from simulated RNA-Seq data using the benchmarked tool
+2. List of genes with information on differentially used poly(A) sites identified from the dataset used for simulation
+
+### Metrics
+
+Based on the input data the following metrics are computed:
+
+1. Sensitivity (TPR, True Positive Rate) = TP/(TP+FN)
+2. FPR (False Positive Rate) = FP/(FP+TN)
+
+TP - true positives - genes identified by the tool and present in the orthogonal dataset  
+FP - false positives - genes identified by the tool and not present in the orthogonal dataset  
+FN - false negatives - genes not identified by the tool but present in the orthogonal dataset  
+TN - true negatives - genes not identified by the tool and not present in the orthogonal dataset
+
+### OpenEBench challenges
+
+The metrics are visualised using 2D scatter plots and barplots, as described in _Plots_ section, which are then used for ranking the participating tools.
+A plot for any given input dataset constitutes a benchmarking challenge as understood in OpenEBench schema.
+Not all plots have to be prepared for each dataset, as described in _Plots_ section.
+All differential usage challenges belong to the same benchmarking event.
+
+## Inputs
+
+| # | Format | Link | Example data |
+  | --- | --- | --- | --- |
+  | 1 | TSV | [Wikipedia][wiki-tsv] | [Link][in1] |
+  | 2 | TSV | [Wikipedia][wiki-tsv] | [Link][in2] |
+
+### Additional info inputs
+  
+#### Input 1
+
+This TSV file contains information on differential usage of poly(A) sites identified from RNA-Seq data by the benchmarked method.  
+Columns:
+
+1. gene ID
+2. significance of differential PAS usage
+
+Column names are not added to the file.
+
+#### Input 2
+
+This TSV file contains information on differential usage of poly(A) sites identified from the orthogonal 3'end-seq dataset.  
+Fields are the same as in Input 1.
+
+## Plots
+
+The results of this benchmark will be visualised in OpenEBench using the following plots:
+
+1. **2D scatter plot** visualizing **TPR and FPR** of identification of genes with differentially used PAS.
+
+**X axis** - FPR  
+**Y axis** - TPR
+
+Input datasets:
+
+- RNA-Seq data compared with 3'end sequencing data
+- Simulated RNA-Seq data compared with dataset used for simulation
+
+Ranking: The best performing tool is the one with the highest TPR combined with lowest FPR (top left part of the plot) and the worst performing tool is the one with the lowest TPR combined with highest FPR (bottom left part of the plot. The plot should be divided into diagonal quartiles based on the distance from optimal performance. Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-left, top-right, bottom-left, bottom-right.
```

Comment:
If you want to change the best corner, you just need to include it in the aggregation jsons, in the  "visualization": {} object, you can specified by: "optimization": "bottom-right" for example. By default, it is  "optimization": "top-right"

Reply:
In `assessment_chart.py` it looks like there are only the two options "top-right" and "bottom-right". Is that true for the website viz as well?

Reply:
Yes, it is. But we can implement also top-left if necessary :)

Reply:
@txellferret would it be easy/quick to implement it?

Alternatively, I can:
- change the plots to Precision-Recall, which would have top-right as the best corner, and this would give us similar information
- swap axes, and then bottom-right would be the best corner - not the standard way of plotting this, but that's the only issue